index.d.ts

```diff
@@ -23,6 +23,12 @@ declare class CssWriter {
   toString(): string;
 }
 
+interface Svelte {
+  compile: any;
+  preprocess: any;
+  major_version: number;
```

Comment:
I think this could just be renamed to `version` and perhaps typed to be an integer, if that's a thing.